policy/main_test.go

```diff
@@ -23,19 +23,36 @@ func TestHandleRequestTPP(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	trust_bundle, err := ioutil.ReadAll(f)
+	trustBundle, err := ioutil.ReadAll(f)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	vcertConnector, err = getConnection(os.Getenv("TPPURL"), os.Getenv("TPPUSER"), os.Getenv("TPPPASSWORD"), "", "", base64.StdEncoding.EncodeToString(trust_bundle))
+	vcertConnector, err = getConnection(os.Getenv("TPPURL"), os.Getenv("TPPUSER"), os.Getenv("TPPPASSWORD"), "", "", "", base64.StdEncoding.EncodeToString(trustBundle))
 	if err != nil {
 		t.Fatal(err)
 	}
-	testHandleRequest(t, os.Getenv("TPPZONE"), "UnexistedZone\\Olololololo", `^[\p{L}\p{N}-_*]+\.example\.com$`)
+	testHandleRequest(t, os.Getenv("TPPZONE"), "UnexistedZone\\Olololololo", ".*")
 }
 
-func testHandleRequest(t *testing.T, zoneName, unexistedZone, checkRegexp string) {
+func TestHandleRequestTPPToken(t *testing.T) {
+	f, err := os.Open(os.Getenv("TRUST_BUNDLE"))
+	if err != nil {
+		t.Fatal(err)
+	}
+	trustBundle, err := ioutil.ReadAll(f)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	vcertConnector, err = getConnection(os.Getenv("TPP_TOKEN_URL"), "", "", os.Getenv("TPP_ACCESS_TOKEN"), os.Getenv("TPP_REFRESH_TOKEN"), "", base64.StdEncoding.EncodeToString(trustBundle))
+	if err != nil {
+		t.Fatal(err)
+	}
+	testHandleRequest(t, os.Getenv("TPPZONE"), "UnexistedZone\\Olololololo", ".*")
```

Comment:
"UnexistedZone" -> "InvalidZone" as you've done elsewhere

Reply:
Changed